src/pages/components/form/style.mdx

```diff
@@ -83,3 +99,32 @@ content and layout of the design. On mobile, forms can only have one column.
 </div>
 
 <Caption>Structure and spacing measurements</Caption>
+
+## AI presence
+
+The following are the unique styles applied to the component when the AI slug is
+present. Unless specified, all other tokens in the component remain the same as
+the non-AI variant.
+
+More information about AI style elements is coming soon.
+
+| Element                    | Property         | Token / Size        |
+| -------------------------- | ---------------- | ------------------- |
+| Form:background            | background-color | `$layer` \*         |
+|                            | box-shadow       | `$ai-drop-shadow`   |
+|                            | inner-shadow     | `$ai-inner-shadow`  |
+| Linear gradient:background | start            | `$ai-aura-start-sm` |
+|                            | stop             | `$ai-aura-stop`     |
+| Linear gradient:border     | top              | `$ai-border-start`  |
+|                            | bottom           | `$ai-border-end`    |
+| AI Slug                    | size             | large               |
+
+<Row>
+<Column colLg={8}>
+
+![Form AI presence](images/form-style-presence.png)
```

Comment:
The 12px spacer is hanging over the top edge here.
<img width="373" alt="image" src="https://github.com/carbon-design-system/carbon-website/assets/11670886/e0332934-d26f-4aff-9ab8-296dac640832">